test/spec/runners/operations/runner.ex

```diff
@@ -16,67 +17,108 @@ defmodule OperationsTestRunner do
         case_dir,
         handler,
-        OperationsTestUtils.resolve_name_from_handler(handler)
+        OperationsTestUtils.resolve_name_from_handler(handler),
+        config
       )
 
-    {:ok, post} = OperationsTestUtils.prepare_test(case_dir, handler, "post")
+    {:ok, post} = OperationsTestUtils.prepare_test(case_dir, handler, "post", config)
 
-    handle_case(testcase.handler, pre, operation, post, case_dir)
+    handle_case(testcase.handler, pre, operation, post, case_dir, config)
   end
 
-  def handle_case("attestation", pre, operation, post, _case_dir),
-    do: OperationsTestAssert.assert_process_attestation(pre, operation, post)
+  def handle_case("attestation", _pre, _operation, _post, _case_dir, _config) do
+    # TODO
+    assert false
+  end
 
-  def handle_case("attester_slashing", pre, operation, post, _case_dir),
-    do: OperationsTestAssert.assert_process_attester_slashing(pre, operation, post)
+  def handle_case("attester_slashing", _pre, _operation, _post, _case_dir, _config) do
+    # TODO
+    assert false
+  end
 
-  def handle_case("block_header", pre, operation, post, _case_dir),
-    do: OperationsTestAssert.assert_process_block_header(pre, operation, post)
+  def handle_case("block_header", _pre, _operation, _post, _case_dir, _config) do
+    # TODO
+    assert false
+  end
 
-  def handle_case("deposit", pre, operation, post, _case_dir),
-    do: OperationsTestAssert.assert_process_deposit(pre, operation, post)
+  def handle_case("deposit", _pre, _operation, _post, _case_dir, _config) do
+    # TODO
+    assert false
+  end
 
-  def handle_case("proposer_slashing", pre, operation, post, _case_dir),
-    do: OperationsTestAssert.assert_process_proposer_slashing(pre, operation, post)
+  def handle_case("proposer_slashing", _pre, _operation, _post, _case_dir, _config) do
+    # TODO
+    assert false
+  end
 
-  def handle_case("voluntary_exit", pre, operation, post, _case_dir),
-    do: OperationsTestAssert.assert_process_voluntary_exit(pre, operation, post)
+  def handle_case("voluntary_exit", _pre, _operation, _post, _case_dir, _config) do
+    # TODO
+    assert false
+  end
 
-  def handle_case("sync_aggregate", pre, operation, post, _case_dir),
-    do: OperationsTestAssert.assert_process_sync_aggregate(pre, operation, post)
+  def handle_case("sync_aggregate", _pre, _operation, _post, _case_dir, _config) do
+    # TODO
+    assert false
+  end
 
-  def handle_case("execution_payload", pre, operation, post, case_dir) do
-    %{execution_valid: execution_valid} =
+  def handle_case("execution_payload", pre, operation, post, case_dir, config) do
+    %{execution_valid: _execution_valid} =
       YamlElixir.read_from_file!(case_dir <> "/execution.yaml")
       |> SpecTestUtils.parse_yaml()
 
-    OperationsTestAssert.assert_process_execution_payload(pre, operation, post, execution_valid)
+    # patch(BeaconChain.StateTransition, :process_execution_payload, )
+
+    assert(post != "no post")
```

Comment:
I changed the code in another PR, so post will be `nil` instead of "no post".

Considering the above, the assertion should be

```
patch(EngineApi, :verify_and_notify_new_payload, fn arg -> execution_valid end)

if post == nil do
   assert  BeaconChain.StateTransition.process_execution_payload(pre, operation, config) == {:error, _foo}

restore(EngineApi)
```

something like this